docs-italia-theme/layouts/breadcrumbs.html

```diff
@@ -58,30 +58,34 @@
       {% else %}
-      <a href="https://bitbucket.org/{{ bitbucket_user }}/{{ bitbucket_repo }}/src/{{ bitbucket_version}}{{ conf_py_path }}{{ pagename }}{{ suffix }}"><i class="fab fa-bitbucket"></i> {{ _('Edit on Bitbucket') }}</a>
+      <a class="Button Button--default u-borderRadius-m u-linkClean u-hidden u-sm-block u-md-block u-lg-block" href="https://bitbucket.org/{{ bitbucket_user }}/{{ bitbucket_repo }}/src/{{ bitbucket_version}}{{ conf_py_path }}{{ pagename }}{{ suffix }}"><i class="fab fa-bitbucket"></i> {{ t.edit_on_bitbucket }}</a>
+      <a class="u-sm-hidden u-md-hidden u-lg-hidden u-linkClean u-color-50" href="https://bitbucket.org/{{ bitbucket_user }}/{{ bitbucket_repo }}/src/{{ bitbucket_version}}{{ conf_py_path }}{{ pagename }}{{ suffix }}">{{ t.edit_on_bitbucket }}</a>
       {% endif %}
       {% elif display_gitlab %}
       {% if check_meta and 'gitlab_url' in meta %}
       <!-- User defined GitLab URL -->
-      <a href="{{ meta['gitlab_url'] }}"><i class="fab fa-gitlab"></i> {{ _('Edit on GitLab') }}</a>
+      <a class="Button Button--default u-borderRadius-m u-linkClean u-hidden u-sm-block u-md-block u-lg-block" href="{{ meta['gitlab_url'] }}"><i class="fab fa-gitlab"></i><span class="u-padding-left-xs">{{ t.edit_on_gitlab }}</span></a>
+      <a class="u-sm-hidden u-md-hidden u-lg-hidden u-linkClean u-color-50" href="{{ meta['gitlab_url'] }}">{{ t.edit_on_gitlab }}</a>
       {% else %}
-      <a href="https://{{ gitlab_host|default("gitlab.com") }}/{{ gitlab_user }}/{{ gitlab_repo }}/blob/{{ gitlab_version }}{{ conf_py_path }}{{ pagename }}{{ suffix }}"><i class="fab fa-gitlab"></i> {{ _('Edit on GitLab') }}</a>
+      <a class="Button Button--default u-borderRadius-m u-linkClean u-hidden u-sm-block u-md-block u-lg-block" href="https://{{ gitlab_host|default("gitlab.com") }}/{{ gitlab_user }}/{{ gitlab_repo }}/blob/{{ gitlab_version }}{{ conf_py_path }}{{ pagename }}{{ suffix }}"><i class="fab fa-gitlab"></i><span class="u-padding-left-xs">{{ t.edit_on_gitlab }}</span></a>
+      <a class="u-sm-hidden u-md-hidden u-lg-hidden u-linkClean u-color-50" href="https://{{ gitlab_host|default("gitlab.com") }}/{{ gitlab_user }}/{{ gitlab_repo }}/blob/{{ gitlab_version }}{{ conf_py_path }}{{ pagename }}{{ suffix }}">{{ t.edit_on_gitlab }}</a>
       {% endif %}
       {% elif show_source and source_url_prefix %}
-      <a href="{{ source_url_prefix }}{{ pagename }}{{ suffix }}">{{ _('Vedi sorgente pagina') }}</a>
+      <a href="{{ source_url_prefix }}{{ pagename }}{{ suffix }}">{{ t.view_page_source }}</a>
       {% elif show_source and has_source and sourcename %}
-      <a href="{{ pathto('_sources/' + sourcename, true)|e }}" rel="nofollow"> {{ _('Vedi sorgente pagina') }}</a>
+      <a href="{{ pathto('_sources/' + sourcename, true)|e }}" rel="nofollow"> {{ t.view_page_source }}</a>
       {% endif %}
```

Comment:
C'è un motivo perché quest'ultimo non ha uno stile a bottone?
```nunjucks
{% elif show_source and source_url_prefix %}
<a class="Button Button--default u-borderRadius-m u-linkClean u-hidden u-sm-block u-md-block u-lg-block" href="{{ source_url_prefix }}{{ pagename }}{{ suffix }}">{{ t.view_page_source }}</a>
<a class="u-sm-hidden u-md-hidden u-lg-hidden u-linkClean u-color-50" href="{{ source_url_prefix }}{{ pagename }}{{ suffix }}">{{ t.view_page_source }}</a>
{% elif show_source and has_source and sourcename %}
<a class="Button Button--default u-borderRadius-m u-linkClean u-hidden u-sm-block u-md-block u-lg-block" href="{{ pathto('_sources/' + sourcename, true)|e }}" rel="nofollow">{{ t.view_page_source }}</a>
<a class="u-sm-hidden u-md-hidden u-lg-hidden u-linkClean u-color-50" href="{{ pathto('_sources/' + sourcename, true)|e }}" rel="nofollow">{{ t.view_page_source }}</a>
{% endif %}
```

Reply:
È un link visualizzato solo in locale quindi non ho implementato il bottone come su readthedocs.